modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Unbound event handlers were incorrectly running when unbound in a previous event handler #TINY-7436
+
```

Comment:
Should we add a `### Changed` and discuss how _adding_ an event handler while the event is firing no longer takes effect immediately?

Reply:
That was never meant to be supported as far as I'm aware, so we don't want a `Changed`. Maybe a `Fixed` but I'm also not convinced it needs to be mentioned here.